### ISSUE TITLE
auditwheel repair: whls not getting compressed for auditwheel 5.1.0 and above?

### ISSUE THREAD

**Issue by lisaong** [pypa/auditwheel#366]
After upgrading to 5.1.0, we noticed that our wheels are not getting compressed, resulting in very large whls after running `auditwheel repair foo.whl`:

```
-rw-r--r-- 1 root root 104093514 Jan  7 01:54 accera-1.2.1.dev17-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.5.1.0.whl
```

auditwheel 5.0.0 was able to compress the whls by roughly 70%:

```
-rw-r--r-- 1 root root  35413658 Jan  7 01:44 accera-1.2.1.dev17-cp38-cp38-manylinux_2_17_x86_64.manylinux2014_x86_64.5.0.0.whl
```

The contents of both files are the same after unzipping, so the binaries themselves are unchanged but the whl does not seem to be compressed when using auditwheel 5.1.0 and above.

```shell
[root@9a0260dd545d wheelhouse]# ls -al accera-5.0.0/
total 101324
drwxr-xr-x 8 root root      4096 Jan  7 01:44 .
drwxr-xr-x 6 root root      4096 Jan  7 01:55 ..
-rw-r--r-- 1 root root       476 Jan  7 01:44 Constants.py
-rw-r--r-- 1 root root      3219 Jan  7 01:44 Debug.py
drwxr-xr-x 2 root root      4096 Jan  7 01:44 hat
-rw-r--r-- 1 root root      1035 Jan  7 01:44 __init__.py
drwxr-xr-x 2 root root      4096 Jan  7 01:44 lang
-rwxr-xr-x 1 root root 103664768 Jan  7 01:44 _lang_python.cpython-38-x86_64-linux-gnu.so
-rw-r--r-- 1 root root     20883 Jan  7 01:44 Package.py
-rw-r--r-- 1 root root      1563 Jan  7 01:44 Parameter.py
drwxr-xr-x 2 root root      4096 Jan  7 01:44 samples
-rw-r--r-- 1 root root      9045 Jan  7 01:44 Targets.py
drwxr-xr-x 2 root root      4096 Jan  7 01:44 test
drwxr-xr-x 3 root root      4096 Jan  7 01:44 tools
drwxr-xr-x 2 root root      4096 Jan  7 01:44 tuning
-rw-r--r-- 1 root root        36 Jan  7 01:44 _version.py


[root@9a0260dd545d wheelhouse]# ls -al accera-5.1.0/
total 101324
drwxr-xr-x 8 root root      4096 Jan  7 01:54 .
drwxr-xr-x 6 root root      4096 Jan  7 01:55 ..
-rw-r--r-- 1 root root       476 Jan  7 01:54 Constants.py
-rw-r--r-- 1 root root      3219 Jan  7 01:54 Debug.py
drwxr-xr-x 2 root root      4096 Jan  7 01:54 hat
-rw-r--r-- 1 root root      1035 Jan  7 01:54 __init__.py
drwxr-xr-x 2 root root      4096 Jan  7 01:54 lang
-rwxr-xr-x 1 root root 103664768 Jan  7 01:54 _lang_python.cpython-38-x86_64-linux-gnu.so
-rw-r--r-- 1 root root     20883 Jan  7 01:54 Package.py
-rw-r--r-- 1 root root      1563 Jan  7 01:54 Parameter.py
drwxr-xr-x 2 root root      4096 Jan  7 01:54 samples
-rw-r--r-- 1 root root      9045 Jan  7 01:54 Targets.py
drwxr-xr-x 2 root root      4096 Jan  7 01:54 test
drwxr-xr-x 3 root root      4096 Jan  7 01:54 tools
drwxr-xr-x 2 root root      4096 Jan  7 01:54 tuning
-rw-r--r-- 1 root root        36 Jan  7 01:54 _version.py
```

**Comment by mayeut** [comment]
thanks for the report. This will be fixed ASAP

**Comment by mayeut** [comment]
Fixed in auditwheel 5.1.2
